lib/backend.js

```diff
@@ -366,7 +363,15 @@ Backend.prototype.fetch = function(agent, index, id, callback) {
     if (err) return callback(err);
     var snapshotProjection = backend._getSnapshotProjection(backend.db, projection);
     var snapshots = [snapshot];
-    backend._sanitizeSnapshots(agent, snapshotProjection, collection, snapshots, backend.SNAPSHOT_TYPES.current, function(err) {
+    var requestContext = {
+      method: 'fetch',
```

Comment:
It would be nice to move these magic strings into an object, similar to the middleware actions. Just to clarify which methods are available, and to make refactoring easy, etc.